scripts/runtime/roxdb.sh

```diff
@@ -58,6 +60,12 @@ validate_and_setup() {
       container="db"
       database="postgres"
       ;;
+    "scanner4")
```

Comment:
```suggestion
    scanner4|scannerv4|scanner-v4)
```

Reply:
Add a few more to the list, good suggestion.